### ISSUE TITLE
Updating to laravel 5.4 errors

### ISSUE THREAD

**Issue by rodzzlessa24** [laracasts/Laravel-5-Generators-Extended#121]
```
Trait 'Illuminate\Console\AppNamespaceDetectorTrait' not found
```

**Comment by i-bajrai** [comment]
Change AppNamespaceDetectorTrait to DetectsApplicationNamespace

**Comment by dansup** [comment]
Use 'dev-master' and run 'composer update' to pull in fix in latest commit 4e9ce5d
